src/Nethermind/Nethermind.Evm/Instructions/EvmInstructions.cs

```diff
@@ -366,7 +366,7 @@ private static bool ChargeAccountAccessGas(ref long gasAvailable, VirtualMachine
             }
 
             // If the account is cold (and not a precompile), charge the cold access cost.
-            if (!vm.CodeInfoRepository.IsPrecompile(address, spec) && vmState.AccessTracker.WarmUp(address))
```

Comment:
Can we just make `CodeInfoRepository.IsPrecompile` work correctly with changed precompiles?

Reply:
I'm not sure about that. It is no longer a precompile as we replaced it with code